exported/localized/de_patch/text/game/items.stringtable

```diff
@@ -9047,7 +9047,7 @@ Des Weiteren solltet ihr nicht vergessen, dass wir NICHT als Plünderer hier sin
     </Entry>
     <Entry>
       <ID>3629</ID>
-      <DefaultText>Hanf</DefaultText>
+      <DefaultText>Hemp</DefaultText>
```

Comment:
Hemp ist Hanf oder was ist hier gemeint?

Reply:
Bah, hätte commit Betreff lesen sollen.

Aber eventuell sollte man den Hund auch übersetzen und die Beschreibung anpassen?

Reply:
Hemp ist der Name des Hundes. Steht so auch in der Textbeschreibung:  "Während Hemp dir folgt, erhältst du einen Bonus auf Verfassung und deine Gruppenmitglieder erhalten einen Bonus auf Tapferkeit."

Reply:
Ja, aber eventuell sollte man den Hund auch Hanf nennen? Kommt halt ein wenig auf den Besitzer an. Hat es einen Grund, dass er Hanf heisst, oder ist das eher ein Zufall...

Gibt ja Leute, die ihren Hund Schnitzel nennen...

Reply:
Ich übernehme es aber erstmal. Kann man sich ja immer noch überlegen.

Reply:
Das ist einfach nur ein Begleittier, das man unterwegs findet. Hemp/Hanf hat keine Hintergrundstory.

Reply:
Ok, in dem Fall lassen wir es so.